src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -458,12 +481,18 @@
     </ProjectReference>
     <Reference Include="Microsoft.Azure.Management.Sql, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.15.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.17.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Storage">
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Storage.2.4.0-preview\lib\net40\Microsoft.Azure.Management.Storage.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.WindowsAzure.Management.Storage">
```

Comment:
Is "Microsoft.Azure.Management.Storage" missing some functionality that "Microsoft.WindowsAzure.Management.Storage" provides? Why are both present at the same time?

Also, we are moving to using SDK's from the common code, [heres the Storage SDK](https://github.com/Azure/azure-powershell/tree/preview/src/Common/Commands.Common.Storage.Management) in the common code.. can you use this instead?

Reply:
No need, Removed.